rpc/common/moc_common_nodeinfo.proto

```diff
@@ -33,6 +33,7 @@ message GPU {
 message OperatingSystem {
 	uint64 operatingsystemsku = 1;
 	OperatingSystemType ostype = 2;
+	string operatingsystembuildnumber = 3;
```

Comment:
This is looking different from the design doc. Is there a reason for the difference?

Reply:
The OS version is what we want to check which is the OS build number. I named it build number to be consistent with wmi property name (the wmi property name is called build number)
Since the type is simply string, there is no need to create another type and define the type as string.

Reply:
Makes sense. We usually align closely to azure-sdk. This makes the code extensisble when azure adds new feature.